utils/build-script

```diff
@@ -999,6 +999,9 @@ def main_preset():
     parser.add_argument(
         "--build-dir",
         help="specify the directory where build artifact should be stored")
+    parser.add_argument(
+        "--swift-profile-instr-use",
+        help="specify the PGO file to use when building swift")
```

Comment:
I don't think this is a good idea. Dmitri deliberately made build-script not accept arguments with presets so that people wouldn't end up building the same preset in different configurations. I'm not sure I agree with that principle, but if we really don't like it we should remove the restriction for all arguments, not add individual arguments in one-off places. (The fact that `--clang-profile-instr-use` doesn't show up here is telling.)

Reply:
I think the restriction of adding additional arguments on top of a preset can provide little value. I think you're right that we should therefore allowing all arguments instead of the selected ones. Will fix in another commit.

Reply:
It was controversial enough at the time Dmitri added it that it's probably worth bringing up on the forums. I'll vote for your change, though.

Reply:
i found myself often wanna slightly tweak an existing preset to either debugging or experimenting something, e.g. comparing with/without PGO enabled. Having such restrictions makes it harder IMO

Reply:
Again, I am in favor of this, but I think it's worth bringing up with a little more visibility.